templates/daemon-optimizer.yaml

```diff
@@ -27,8 +27,8 @@ spec:
         path: /app/start_proxy.py
         raw:
           data: |
-            from zquantum.core.optimizer.optimize import optimize_variational_circuit_with_proxy
-            from zquantum.core.optimizer.proxy import Client
+            from zquantum.optimizers.daemon_optimizer.optimize import optimize_variational_circuit_with_proxy
```

Comment:
If we can remove `optimize_variational_circuit_with_proxy` then we will need to update this template.